bugsnag-android-performance/src/main/kotlin/com/bugsnag/android/performance/internal/instrumentation/ActivityLifecycleInstrumentation.kt

```diff
@@ -118,9 +118,7 @@ internal abstract class AbstractActivityLifecycleInstrumentation(
         onViewLoadLeak(activity)
     }
 
-    override fun onActivityStopped(activity: Activity) {
-        onViewLoadLeak(activity)
-    }
+    override fun onActivityStopped(activity: Activity) {}
```

Comment:
using ` = Unit` removes the need for the `detekt-baseline` change

```suggestion
    override fun onActivityStopped(activity: Activity) = Unit
```